src/components/Icon/ExpensifyIconLoader.ts

```diff
@@ -61,6 +71,15 @@ function loadExpensifyIcon(iconName: ExpensifyIconName): Promise<{default: IconA
         });
 }
 
-export {loadExpensifyIcon, loadExpensifyIconsChunk};
+/**
+ * Get the cached ExpensifyIcons chunk synchronously
+ * Returns null if the chunk hasn't been loaded yet
+ * Use this to avoid Promise microtask delay when chunk is already loaded
+ */
+function getExpensifyIconsChunkSync(): ExpensifyIconsChunk | null {
```

Comment:
```suggestion
function getExpensifyIconsChunk(): ExpensifyIconsChunk | null {
```
Isn't `getExpensifyIconsChunk` enough to describe?